tests/test_pint_units.py

```diff
@@ -0,0 +1,97 @@
+from math import pi
+
+import pytest
+from pint import Quantity
+
+from pnpq.units import ureg
+
+
+@pytest.mark.parametrize(
+    "test_mpc320_step, expected_angle",
+    [
+        (-100, -100 * (170 / 1370)),
+        (0, 0),
+        (100, 100 * (170 / 1370)),
+    ],
+)
+def test_mpc320_step_to_angle_conversion(
+    test_mpc320_step: float, expected_angle: float
+) -> None:
+
+    angle = (test_mpc320_step * ureg.mpc320_step).to("degrees").magnitude
+    assert angle == pytest.approx(expected_angle)
+
+
+@pytest.mark.parametrize(
+    "test_angle, expected_mpc320_step",
+    [
+        (-100, -100 / (170 / 1370)),
+        (0, 0),
+        (100, 100 / (170 / 1370)),
+    ],
+)
+def test_angle_to_mpc320_step_conversion(
+    test_angle: float, expected_mpc320_step: float
+) -> None:
+
+    mpc320_step = (test_angle * ureg.degree).to("mpc320_step").magnitude
```

Comment:
This wouldn't work for mpc320_step, because we still do float calculations with it. And for reasons explained in my comment up there, I can't seem to get the transformations to work for mpc320_steps either. So we can't add this right now. Sad.